lib/active_encode/engine_adapters/ffmpeg_adapter.rb

```diff
@@ -98,10 +98,17 @@ def find(id, opts = {})
 
       # Cancel ongoing encode using pid file
       def cancel(id)
-        pid = get_pid(id)
-        Process.kill 'SIGTERM', pid.to_i
-
-        find id
+        encode = find id
+        if encode.running?
+          pid = get_pid(id)
+          Process.kill 'SIGTERM', pid.to_i
+          encode = find id
+        end
+        encode
+      rescue Errno::ESRCH
```

Comment:
Is this indentation conventional?

Reply:
Yes, see the second example here: https://docs.ruby-lang.org/en/2.4.0/syntax/exceptions_rdoc.html